tests/FixtureBuilder/Payload.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tuf\Tests\FixtureBuilder;
+
+use Tuf\CanonicalJsonTrait;
+
+/**
+ * A payload is a builder for a server-side TUF metadata file.
+ *
+ * A payload represents a single role in the TUF repository. This could be one
+ * of the four top-level roles (root, timestamp, snapshot, and targets), or it
+ * could be a delegated targets role at any level of nesting.
+ *
+ * Every payload has relationships to other payloads. There are two kinds
+ * of relationships:
+ *
+ * - A "parent" relationship is for a payload that enumerates other payloads.
```

Comment:
Should the docs for parent and key ring just `@param` docs for `__construct`?

Reply:
I thought about that, but I don't think that's really the best place for it. This class-level doc block is more general, and needs to explain the overall choices of how these classes are meant to fit together.